ledger-core/virtual/integration/constructor_api_test.go

```diff
@@ -17,7 +17,7 @@ import (
 func Test_API_Create(t *testing.T) {
 	t.Log("C4837")
 
-	server, ctx := utils.NewServer(nil, t)
+	server, ctx := utils.NewServer(nil, t, true)
```

Comment:
true arg doesn't read here well
may I suggest to have another set of NewServerXXX() methods pls? or a builder

Reply:
fixed